src/renderer/views/multiplayer/custom.vue

```diff
@@ -179,7 +182,18 @@ let active = true;
 
 const abortController = new AbortController();
 
+const watchForLobbyJoin = api.comms.onResponse("s.lobby.updated_client_battlestatus").add(() => {
+    loading.value = false;
+});
+
+const watchForLobbyJoinFailure = api.comms.onResponse("s.lobby.join").add((data) => {
+    data.result === "failure" || data.reason === "Battle locked" ? (loading.value = false) : null;
```

Comment:
Same as other comment, would prefer this to be an `if` block as it's more readable:
```typescript
if (data.result === "failure" || data.reason === "Battle locked") {
    loading.value = false;
}
```

Reply:
changed